CMakeLists.txt

```diff
@@ -413,10 +418,12 @@ ENDIF (HAVE_LINKER_AS_NEEDED)
 ADD_SUBDIRECTORY (src)
 ADD_SUBDIRECTORY (ext)
 ADD_SUBDIRECTORY (tools)
+IF (NOT WIN32)
 IF (ENABLE_PERL OR ENABLE_PYTHON OR ENABLE_RUBY OR ENABLE_TCL)
     ADD_SUBDIRECTORY (bindings)
 ENDIF (ENABLE_PERL OR ENABLE_PYTHON OR ENABLE_RUBY OR ENABLE_TCL)
 ADD_SUBDIRECTORY (examples)
+ENDIF (NOT WIN32)
```

Comment:
This shouldn't be necessary, since bindings aren't enabled by default anyway. It'd probably be better for examples to get a dedicated configuration switch rather than wiring this into a `NOT WIN32` condition.

Reply:
done